doc/api.md

```diff
@@ -65,6 +81,99 @@ var api = new JitsiMeetExternalAPI(domain, options);
 
 ### Controlling the embedded Jitsi Meet Conference
 
+You can control the available devices  with the following methods of `JitsiMeetExternalAPI` instance:
+* **getAvailableDevices** - Retrieve a list of available devices.
+
+```javascript
+api.getAvailableDevices().then(function(devices) {
```

Comment:
would it make sense to group all these methods in some namespace like `api.mediaDevices.getAvailableDevices()` and such?

Reply:
I don't have any preferences about this. If you would like it better I will change it. 

@saghul Do you want me to change it?

Reply:
After discussing with @saghul we come up with 2 approaches to implement the namespace:
1) add the methods to the namespace but leave the options in the constructor for the initial devices to be set. IMHO this will be a little inconsistent.
2) implement a way to create the iframe outside of the constructor. This way we can remove the options of the constructor and use the methods from the namespace for the initial setting of devices. IMHO this will complicate the API flow for the consumers.

So I prefer to leave it as it is!

Reply:
👍